lib/verk/sorted_set.ex

```diff
@@ -68,6 +68,36 @@ defmodule Verk.SortedSet do
   end
 
   @doc """
+  Lists jobs from `start` to `stop` along with the item scores
+  """
+  @spec range_with_score(String.t, integer, integer, GenServer.server)
+    :: {:ok, [{Verk.Job.T, integer}]} | {:error, Redix.Error.t}
+  def range_with_score(key, start \\ 0, stop \\ -1, redis) do
+    case Redix.command(redis, ["ZRANGE", key, start, stop, "WITHSCORES"]) do
+      {:ok, jobs} ->
+        # The Redis returned list alternates, [<job>, <job score>, ...].
+        scores =
+          jobs
+          |> Enum.drop_every(2)
+          |> Enum.map(fn score -> String.to_integer(score) end)
+        decoded_jobs =
+          jobs
+          |> Enum.take_every(2)
+          |> Enum.map(fn job -> Job.decode!(job) end)
+        {:ok, Enum.zip(decoded_jobs, scores)}
```

Comment:
This is great! I was trying to find something like this, but didn't notice that method :)